applets/wncklet/window-list.c

```diff
@@ -1060,6 +1104,16 @@ static void setup_dialog(GtkBuilder* builder, TasklistData* tasklist)
 	g_signal_connect(G_OBJECT(tasklist->auto_group_radio), "toggled", (GCallback) group_windows_toggled, tasklist);
 	g_signal_connect(G_OBJECT(tasklist->always_group_radio), "toggled", (GCallback) group_windows_toggled, tasklist);
 
+	/* Mouse Scroll: */
+	g_settings_bind_with_mapping (tasklist->settings,
```

Comment:
why don't you use `g_settings_bind` without mapping?

Reply:
Has been updated